style.css

```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;700&display=swap');
+
+* {
+  border: 0;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.menu-button {
+  display: flex;
+  justify-content: end;
+  margin-right: 10%;
+}
+
+.main-container {
+  background-image: url(media/header-illsutration-mobile.svg);
+  height: 700px;
+}
```

Comment:
- [ ] Please I highly recommend using CSS Flexbox for the `main-container` class as it is part of the project requirements. 😄

Reply:
@braincee required change made.
<img width="407" alt="image" src="https://user-images.githubusercontent.com/118947090/227563337-0e55be77-3ca5-4382-9d59-a10d4cb64e58.png">